dojo_plugin/api/v1/dojos.py

```diff
@@ -108,18 +120,30 @@ def get(self, dojo):
             dict(id=module.id,
                  name=module.name,
                  description=module.description,
-                 challenges=[
-                    dict(id=challenge.id,
-                         name=challenge.name,
-                         required=challenge.required,
-                         description=challenge.description)
-                    for challenge in (module.visible_challenges() if not is_dojo_admin
-                                      else module.challenges)
-                 ])
```

Comment:
This is breaking testcases, and a number of clients expect this data to exist like this. Probably we should just bring it back in its original form, but we can also bring in unified_items.